ruby/.ruby-style.yml

```diff
@@ -85,6 +90,9 @@ Style/BlockDelimiters:
                 Prefer {...} over do...end for single-line blocks.
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#single-line-blocks'
   Enabled: true
+  Exclude:
```

Comment:
The changes here were promoted from backend's personal config. (https://github.com/zestyzesty/backend/pull/5849/files)